constructorio-client/src/main/java/io/constructor/client/ConstructorIO.java

```diff
@@ -13,15 +13,11 @@
 
 import com.google.gson.Gson;
 
+import io.constructor.client.models.*;
```

Comment:
Was this intentional or did the your IDE organize stuff?

Reply:
Ide did it by itself! I just reverted it.